### DISCUSSION TITLE
no iv parameter in des-decryption

### DISCUSSION THREAD

**Discussion by austond** [he3-app/he3-feedback#314] [Bug Report]
### Version
des-decryption 1.0.4

### Environment
Portal

### What is expected?
when using DES encryption, it should take passphrase and iv as input, like what is in AES encryption

### What is actually happening?
only passphrase available as input

**Comment by zhenweiwang1990** [comment]
We'd like if you may contribute this feature: https://github.com/he3-app/tools-example/tree/main/des-decryption
